test/uk/gov/hmrc/customs/emailfrontend/utils/ViewTestHelper.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.emailfrontend.utils
+
+import org.jsoup.nodes.Document
+import org.scalatest.Assertion
+import org.scalatest.matchers.must.Matchers.mustBe
+
+trait ViewTestHelper extends SpecBase {
+
+  def shouldContainCorrectServiceUrls(viewDoc: String, routes: String): Assertion = {
```

Comment:
Better to save urls as name for the second parameter as it is better name that aligns with the method.